manifest.json

```diff
@@ -3,12 +3,12 @@
   "name": "Free Medium",
   "version": "0.0.1",
   "description": "This browser extension allows users to read Medium articles for free.",
-   "permissions": ["scripting"],
+   "permissions": ["scripting", "cookies"],
```

Comment:
Needed access to the cookies permission, since I realized that cookies are being used to determine if you've surpassed your monthly member-only article limit.